packages/cli/package.json

```diff
@@ -21,29 +21,32 @@
     "@getpochi/livekit": "workspace:*",
     "@getpochi/tools": "workspace:*",
     "@getpochi/vendor-claude-code": "workspace:*",
+    "@getpochi/vendor-codex": "workspace:*",
     "@getpochi/vendor-gemini-cli": "workspace:*",
+    "@getpochi/vendor-github-copilot": "workspace:*",
     "@getpochi/vendor-pochi": "workspace:*",
-    "@getpochi/vendor-codex": "workspace:*",
     "@getpochi/vendor-qwen-code": "workspace:*",
     "@inquirer/select": "^4.3.2",
-    "@getpochi/vendor-github-copilot": "workspace:*",
     "@livestore/adapter-node": "catalog:",
     "@livestore/livestore": "catalog:",
     "@livestore/sync-cf": "catalog:",
     "@livestore/utils": "catalog:",
     "@livestore/wa-sqlite": "catalog:",
     "@preact/signals-core": "^1.6.2",
-    "@types/omelette": "^0.4.5",
     "@types/semver": "^7.5.8",
+    "@types/tabtab": "^3.0.4",
     "ai": "catalog:",
     "chalk": "^5.3.0",
     "commander": "^14.0.0",
     "jose": "catalog:",
     "listr2": "^9.0.3",
     "node-machine-id": "catalog:",
-    "omelette": "^0.4.17",
     "ora": "^8.2.0",
     "remeda": "catalog:",
+    "semver": "^7.6.3"
+  },
+  "dependencies": {
+    "tabtab": "^3.0.2",
```

Comment:
this should also go to dev dependencies (all deps of cli shall be dev dependencies, as we do bundling)